server/db/migration/migrations/sqls/20211111110342-move-to-legacy-schema-up.sql

```diff
@@ -0,0 +1,7 @@
+create schema _legacy;
+
+alter table public._legacy_assessment set schema _legacy;
+alter table public._legacy_fra_user set schema _legacy;
+alter table public._legacy_odp set schema _legacy;
+alter table public._legacy_odp_class set schema _legacy;
+alter table public._legacy_odp_version set schema _legacy;

```

Comment:
can we remove _legacy prefix from these tables now that they are in _legacy schema?